cysync/src/pages/mainApp/sidebar/wallet/send/generalComponents/CustomSlider.tsx

```diff
@@ -146,7 +146,8 @@ const CustomSlider: React.FC<CustomSliderProps> = ({
     handleTransactionFeeChangeSlider(v);
   };
 
-  const stepValue = step ?? (mediumFee < 5 ? 0.1 : 0.2);
+  const stepValue =
+    step ?? (mediumFee < 5 ? (mediumFee < 0.1 ? 0.00001 : 0.1) : 0.2);
```

Comment:
Can we have a logic which can define the step from medium fees?

Reply:
added. now using 1% as step value for slider